code/src/java/plugin/exporttokens/SpecialAbilityToken.java

```diff
@@ -50,52 +48,15 @@ public String getToken(String tokenSource, PlayerCharacter pc, ExportHandler eh)
 			i = Integer.parseInt(aTok.nextToken());
 		}
 
-		if (aTok.hasMoreTokens())
-		{
-			String subToken = aTok.nextToken();
-
-			if ("DESCRIPTION".equals(subToken))
-			{
-				return getDescriptionToken(pc, i);
```

Comment:
Technically, this is not logically equivalent, so it needs data (or perhaps OS) approval for a change.  Logical equivalence would return "" in the case of DESCRIPTION.

Reply:
Yeah, gonna say no here. 

Unless I'm mistaken, that is referencing DESC which is a multi-tag allowed token/tag in all our data. (Token has next, which means it appends, not replaces, unlike ASPECT/QUALITY/INFO), two other tags mimic that behavior - BENEFIT & SPROP.

If you can prove it's not going to break anything, then I might say yes - I have no idea about ROG, but export tokens, we should be very very clear before we start pulling anything out that might be used for outside support - old CMP/Homebrew situations.

Reply:
As far as I could tell this only affects `DESCRIPTION` if `isROG` is ever true. This can only happen if people set this manually.

Reply:
Restored the "" return